src/shared/components/dateRangePicker/DateRangePicker.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import DateRangePicker from './DateRangePicker'
+import {convertTimeRangeToCustom} from 'src/shared/utils/duration'
+import {pastHourTimeRange} from 'src/shared/constants/timeRanges'
+import {renderWithReduxAndRouter} from 'src/mockState'
+
+import {screen} from '@testing-library/dom'
+import {fireEvent} from '@testing-library/react'
+
+jest.useFakeTimers().setSystemTime(new Date('2022-03-10 20:00:00').getTime())
+
+describe('Date Range Picker', function() {
+  test('should have input values in the correct format after user changes the date selection', () => {
```

Comment:
if you use `it` instead of `test`, it's in theory, supposed to read like a sentence

```ts
it('should have input values in the correct format after user changes the date selection')
```

is meant to be read like

>it should have input values in the correct format after user changes the date selection